turbo/static/turbo/js/turbo-django.js

```diff
@@ -7,44 +7,50 @@
   );
 
   class TurboChannelsStreamSource extends HTMLElement {
-    static counter = 0;
-    request_id;
-
     constructor() {
       super();
-      this.request_id = TurboChannelsStreamSource.counter++;
     }
 
     async connectedCallback() {
       Turbo.connectStreamSource(this);
-
-      socket.addEventListener("open", (e) => {
-        socket.send(
-          JSON.stringify({ request_id: this.request_id, type: "subscribe", ...this.subscription })
-        );
-      });
+      if (socket.readyState !== WebSocket.OPEN) {
+        socket.addEventListener("open", (e) => {
+          this.subscribe();
+        });
+      } else {
+        this.subscribe();
+      }
```

Comment:
This is the same fix @JulianFeinauer made in his PR.

Reply:
Let's hope `git` is kind to us on the rebase 😄